From: Carlos Maddela <e7appew@gmail.com>
Date: Mon, 28 Aug 2017 00:28:01 +1000
Subject: Update Makefile.am files for Automake 1.15.

Description: Update Makefile.am files for Automake 1.15.
 Use AM_CPPFLAGS in preference to INCLUDES.
Author: Carlos Maddela <e7appew@gmail.com>
Last-Update: 2017-08-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 tests/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 55330a1..4bf8d46 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -2,7 +2,7 @@ check_PROGRAMS = oauthexample oauthdatapost tcwiki tceran tcother oauthtest oaut
 ACLOCAL_AMFLAGS= -I m4
 
 OAUTHDIR =../src
-INCLUDES = -I$(srcdir)/$(OAUTHDIR)
+AM_CPPFLAGS = -I$(srcdir)/$(OAUTHDIR)
 MYCFLAGS = @LIBOAUTH_CFLAGS@ @HASH_CFLAGS@ @CURL_CFLAGS@ 
 MYLDADD  = $(OAUTHDIR)/liboauth.la
 LIBS     = -lm @HASH_LIBS@ @CURL_LIBS@ @LIBS@
